default/stringtables/en.txt

```diff
@@ -124,6 +124,9 @@ Back
 LAST
 Last
 
+DIFFICULTY
+Difficulty
+
```

Comment:
Can this label be renamed\titled to avoid future confusion? (`Newbie Bonus`?)
I was under the assumption AI Aggression is not intended as a difficulty setting (although used that way sometimes due to lack of options).